Rakefile

```diff
@@ -54,12 +54,10 @@ task :development_app do
     ["development_app", "--path", "..", "--recreate_db"]
   )
 
-  Dir.chdir("#{__dir__}/development_app") do
```

Comment:
Why is this innecessary? If we remove this, won't `bundle exec rails generate decidim:demo` be run inside the main `decidim` folder? 😕

Reply:
Apparently the `AppGenerator` switches folder and don't go back, so we're already at the app folder at this time.

However, the commit adds nothing and harms readability. Removing it!